releasenotes/notes/Added-random-circuit-from-graph-95c22eeabdea89d0.yaml

```diff
@@ -0,0 +1,35 @@
+---
+features_circuits:
+  - |
+    The function :func:`~qiskit.circuit.random.utils.random_circuit_from_graph` adds a new feature
+    of generating random circuits with interaction graph.
+    User can attach a float value indicating the probability of getting selected as a metadata to
+    the edge of the graph generated. If all the probabilities is passed as `None`, then the probability
+    of each qubit-pair of getting selected is set to 1/N. (where N is the number of edges in the interaction_graph passed in)
+    If float values are present as probabilities but some are None, this will raise a ValueError.
+
+    Arguments accepted by the function are
+    :code:`interaction_graph` Takes in the interaction graph
```

Comment:
Describe what the interaction graph is here

Reply:
Done :)